### PR TITLE
feat(frontend): Implement API endpoint to view fees

### PR DESCRIPTION
**Pull request by shelegdmitriy**
Resolves: #627 

### API Structure

- `/api/fees-of-previous-day-utc`
    Result:

    ```
    {
      date: "2021-08-01",
      collected_fee_in_yoctonear: "15795158391608358300000000"
    }
    ```

- `/api/fees-of-previous-7-days-utc`
    Result:
    ```
    [
      {
        date: "2021-07-26",
        collected_fee_in_yoctonear: "20207612154389927000000000"
      },
      {
        date: "2021-07-27",
        collected_fee_in_yoctonear: "25956649332088157100000000"
      },
      {
        date: "2021-07-28",
        collected_fee_in_yoctonear: "21533636619634191000000000"
      },
      {
        date: "2021-07-29",
        collected_fee_in_yoctonear: "17509630681688307719987884"
      },
      {
        date: "2021-07-30",
        collected_fee_in_yoctonear: "23866291880366084361780368"
      },
      {
        date: "2021-07-31",
        collected_fee_in_yoctonear: "18474313503193355600000000"
      },
      {
        date: "2021-08-01",
        collected_fee_in_yoctonear: "15795401186391479100000000"
      }
    ]
    ```

### CODE REVIEW

**Comment by shelegdmitriy** [comment]
For now there is three endpoints:

- '/api/fee-by-day?days=<number>' - which returns dayTime and fee per that day;
- 'api/fee-per-day' - which returns only totalFeeInYoctoNear by one day;
- 'api/fee-per-week' =  which returns only totalFeeInYoctoNear by 7 days;

I'll appreciate any comments of this

**Comment by frol** [comment]
Here is the requirement from [CryptoFees](https://docs.google.com/forms/d/e/1FAIpQLScLF8KcLpYsxzNl7Jv5AVe0D13ADXey7dcUb85N9Yw5LVNbmg/formResponse):

> Is there a reliable source of data for the fees collected in the previous day and 7 days (UTC time)

@shelegdmitriy I suggest we avoid unnecessary implementations; let's have:

* `/api/fees-of-previous-day-utc`
  Return value:

  ```json
  {
    date: "2021-08-02",
    collected_fee_in_yoctonear: 1
  }
  ```
* `/api/fees-of-previous-7-days-utc`
  Return value:

  ```json
  [
    {
      date: "2021-08-01",
      collected_fees_in_yoctonear: 1
    },
    {
      date: "2021-08-02",
      collected_fees_in_yoctonear: 1
    },
    ...
  ]
  ```

P.S. The other API endpoints already use snake_case naming convention (I choose to use it for JSON fields naming a while ago, so let's stick to it)

File: backend/src/db-utils.js [hunk 1]
```diff
@@ -560,6 +560,27 @@ const getLastYesterdayBlock = async (timestamp) => {
...
+  return await queryRows(
+    [
+      `SELECT
+        DATE_TRUNC('day', TO_TIMESTAMP(DIV(eo.executed_in_block_timestamp, 1000*1000*1000))) AS date,
+        SUM(transactions.receipt_conversion_tokens_burnt + eo.tokens_burnt) AS fee
+      FROM transactions
+      LEFT OUTER JOIN receipts on receipts.originated_from_transaction_hash = transactions.transaction_hash
+      JOIN execution_outcomes eo on eo.receipt_id = receipts.receipt_id
+      WHERE
+        block_timestamp > (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '${days} day')) AS bigint) * 1000 * 1000 * 1000)
```
Comment (frol) [inline backend/src/db-utils.js:574]: Just a minor tweak:

```suggestion
        block_timestamp >= (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '${days} day')) AS bigint) * 1000 * 1000 * 1000)
```

**Review by frol — APPROVED** [review]
LGTM! Congrats on your public API development achievement unlocked :smile:

**Pull request merged**